pyodide-build/pyodide_build/__main__.py

```diff
@@ -35,15 +35,12 @@ def make_parser() -> argparse.ArgumentParser:
 
 def main():
     if not os.environ.get("__LOADED_PYODIDE_ENV"):
-        from .common import get_hostsitepackages
-
-        PYODIDE_ROOT = str(pathlib.Path(__file__).parents[2].resolve())
-        os.environ["PYODIDE_ROOT"] = PYODIDE_ROOT
+        if "PYODIDE_ROOT" not in os.environ:
+            os.environ["PYODIDE_ROOT"] = str(pathlib.Path(os.getcwd()))
```

Comment:
I think putting `Path(__file__).parents[2]` here would be better. Or throw an error here and require people to `source pyodide_env.sh` before running this command.

Otherwise this would break `pyodide-build buildpkg meta.yaml` in `packages/some_pkg` would break because `pyodide-build` would decide that `packages/some_pkg` is the root directory.

Reply:
> Otherwise this would break `pyodide-build buildpkg meta.yaml` in `packages/some_pkg` would break because `pyodide-build` would decide that `packages/some_pkg` is the root directory.

Okay, I'll go with raising an error when `PYODIDE_ROOT` is not explicitly set for now.

My intention for the future is that we introduce a new config file similar to `packages.json` in node or `Cargo.toml` in rust which replaces the current `Makefile.env` and `pyodide_env.sh`, then our new CLI detects that file to ensure that we are in the right directory.

Reply:
Right. Until we get there I would prefer if try not to break the ways I am currently using these tools. Though of course it's fine if it is too hard to avoid.